packages/react/package.json

```diff
@@ -36,6 +36,7 @@
     "@vitejs/plugin-react": "^4.0.0",
     "eslint-plugin-react": "7.31.11",
     "eslint-plugin-react-hooks": "4.6.0",
+    "jsdom": "^22.1.0",
```

Comment:
any idea why this suddenly appeared?

Reply:
my assumption is that it was somehow available because some dependency of the network package